api/src/main/java/eu/darkbot/api/extensions/IssueHandler.java

```diff
@@ -0,0 +1,23 @@
+package eu.darkbot.api.extensions;
+
+import java.util.Set;
+
+public interface IssueHandler {
+
+    Set<? extends Issue> getIssueSet();
+
+    void addInfo(String message, String description);
+    void addWarning(String message, String description);
+    void addFailure(String message, String description);
```

Comment:
probably add an `addIssue(level, message, description)` and default these 3 to that, avoiding the need to implement 3 methods